scripts/2.mutation-classifier.py

```diff
@@ -181,31 +188,30 @@ def select_feature_set_columns(X, feature_set):
                 for mod, pipeline in pipeline_definitions.items()}
 
 
-# In[14]:
-
-st = time.perf_counter()
+# In[13]:
 
 # Fit the models
 for model, pipeline in cv_pipelines.items():
     print('Fitting CV for model: {0}'.format(model))
+    start_time = time.perf_counter()
     pipeline.fit(X=X_train, y=y_train)
-    
-et = time.perf_counter()
-print('Time to fit models: {0}'.format(str(datetime.timedelta(seconds=et-st))))
+    end_time = time.perf_counter()
+    elapsed = datetime.timedelta(seconds=end_time - start_time)
+    print('\truntime: {}'.format(elapsed))
 
 
-# In[15]:
+# In[14]:
 
 # Best Parameters
 for model, pipeline in cv_pipelines.items():
-    print('{0}: {1:.3%}'.format(model, pipeline.best_score_))
-
+    print('#', model)
     print(pipeline.best_params_)
+    print('cv_auroc = {:.3%}'.format(pipeline.best_score_))
 
```

Comment:
Great observation. @rdvelazquez it's possible this could happen from package upgrades. But I think the most likely culprit is the randomness in `GridSearchCV` is not controlled. See https://github.com/scikit-learn/scikit-learn/issues/2046.

Possible solutions:

1. set a random seed for numpy
2. construct a  [`StratifiedKFold`](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.StratifiedKFold.html#sklearn.model_selection.StratifiedKFold) object with a set `random_state`

I'm leaning towards option 2. I wonder if right now each parameter combination is getting evaluated on different CV splits. I don't see any reason not to reuse CV folds. In fact, it may make for a more consistent comparison.